sci-electronics/iverilog/iverilog-10.3.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="A Verilog simulation and synthesis tool"
+HOMEPAGE="http://iverilog.icarus.com/"
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="examples"
+
+RDEPEND="
+	app-arch/bzip2
```

Comment:
you do not really need this dependency.

Reply:
> you do not really need this dependency.

Thank you!

It comes from https://gitweb.gentoo.org/repo/gentoo.git/tree/sci-electronics/iverilog/iverilog-10.2.ebuild
but it should installed by default.
I removed this dependency.